bskyembed/src/components/post.tsx

```diff
@@ -123,7 +123,8 @@ function PostContent({record}: {record: AppBskyFeedPost.Record | null}) {
         <Link
           key={counter}
           href={segment.link.uri}
-          className="text-blue-400 hover:underline">
+          className="text-blue-400 hover:underline"
+          disableTracking={!segment.link.uri.startsWith('https://bsky.app')}>
```

Comment:
is `segment.link.uri` guaranteed to exist?

Reply:
Good shout, updated to use `validateLink` which is what the main app's richtext does